main.py

```diff
@@ -1 +1,179 @@
-print("hello world ")
\ No newline at end of file
+import random
+
+EMPTY_CELL = "."
+FLOWER = "F"
+WEED = "W"
+
+def plant_random(garden , amount , item , row , col):
+    for i in range(amount):
+        while True : 
+            coordi_flower_x = random.randint(0, row - 1)
+            coordi_flower_y = random.randint(0, col - 1)
+            if garden[coordi_flower_x][coordi_flower_y] == EMPTY_CELL:
+                garden[coordi_flower_x][coordi_flower_y] = item
+                break
+    return garden
```

Comment:
We don't need to return the garden here. When you update a list in place (without deep copy) you don't need to return it.